src/App.js

```diff
@@ -1,10 +1,23 @@
+/* eslint-disable  */
```

Comment:
- [ ] Please do not disable Linters. These are tools that help to flag programming bugs. Let's fix these errors.